src/main/java/com/epam/prd/jjop/helpers/objects/StreamOfIntegersCreator.java

```diff
@@ -0,0 +1,43 @@
+package com.epam.prd.jjop.helpers.objects;
```

Comment:
```suggestion
package com.epam.prd.jjop.helpers.streams;
```
This is not about objects/primitives but the creation of the streams.

Reply:
The same remark below